org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WPAttributeDialog.java

```diff
@@ -1185,7 +1185,7 @@ else if (displayType == DisplayType.Image
 				valueLable = editor.getDisplay();

```

Comment:
looks like there's an existing typo here (valueLable instead of valueLabel), would be nice to fix it as well.

Reply:
Yes, good case. Will fix.